__tests__/app.test.js

```diff
@@ -0,0 +1,55 @@
+const request = require('supertest')
+const app = require('../app')
+const seed = require('../db/seeds/seed')
+const db = require('../db/connection')
+const data = require('../db/data/test-data/index')
+
+beforeEach(() => {
+    return seed(data)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe('/api/topics', () => {
+    describe('GET:', () => {
+        describe('status 200:', () => {
+            test('responds with array of topics with slug and decription', () => {
+                return request(app)
+                    .get('/api/topics')
+                    .expect(200)
+                    .then((result) => {
+                        const topics = result.body.topics
+                        topics.map((topic)=> {
```

Comment:
2 things:
Is a map what we want or would a forEach have been more suitable?
Can we check the length, as if the length===0 the test would default pass